persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/client/ClientDAOImpl.kt

```diff
@@ -182,4 +183,8 @@ internal class ClientDAOImpl internal constructor(
             .groupBy { it.userId }
     }
 
+    override suspend fun recipientByUserID(ids: List<UserIDEntity>): Map<QualifiedIDEntity, List<Client>> {
+        TODO("Not yet implemented")
```

Comment:
This needs to implemented, no?

Reply:
ohhh my bad this need to be removed since something similar already exsists `ClientDAO.recipientsIfTheyArePartOfConversation()`